File: Python/plus-minus.py
def plusMinus(arr):
    p = n = z =0
    size = len(arr)
    for num in arr:
        if num>0:
            p+=1
        elif num<0:
            n+=1
        else:
            z+=1

    print(f"{p/size:06f}")
    print(f"{n/size:06f}")
    print(f"{z/size:06f}")

File: Java/plus-minus.java
class Result {
    public static void plusMinus(List<Integer> arr) {
        int p = 0;
        int n = 0;
        int z = 0;
        int size_arr = arr.size();
        for(int i = 0; i< size_arr; i++){
            int val = arr.get(i);
            if(val>0){
                p++;
            }
            else if(val<0){
                n++;
            }
            else{
                z++;
            }
        }
        System.out.printf("%.6f%n",(float)p/(float)size_arr);
        System.out.printf("%.6f%n",(float)n/(float)size_arr);
        System.out.printf("%.6f%n",(float)z/(float)size_arr);
    }
}